cypress/e2e/the-internet/the-internet-tests.cy.js

```diff
@@ -287,13 +287,13 @@ describe("The Internet Test Suite", () => {
 		}
 	});
 
-	it("Tests Infinite Scroll", () => {
+	it.only("Tests Infinite Scroll", () => {
```

Comment:
only keyword here

Reply:
:(

Reply:
This new commit again has the only keyword

Reply:
:)